tests/application-test.js

```diff
@@ -7,11 +7,19 @@ describe('Box.Application', function() {
 
 	'use strict';
 
+	var $ = jQuery.noConflict(true);
 	var sandbox = sinon.sandbox.create();
 
 	var testModule,
 		testModule2,
-		nestedModule;
+		nestedModule,
+		customEvent;
+
+	var dispatch = function(selector) {
+		customEvent = document.createEvent('MouseEvent');
```

Comment:
this is a dilemma really:

1. if you want to the test to run in IE8, then jQuery is needed.

2. but if you want to test `addEventListener`, then jQuery must not in the global, at least with current fallback design.